website/docs/monitoring.md

```diff
@@ -0,0 +1,89 @@
+
+The details of these functions are listed below :
+
+### `setMonitoringUser`
+
+``` typescript
+setMonitoringUser(id: string): void;
+```
+
+This function will add current user to monitoring application so that whenever users measure any performance or set a custom measurements in the monitoring, at that time we will have a all the transaction under current user. It will help us to identify the transaction based on user.
+
+### `setMonitoringTag`
+
+``` typescript
+setMonitoringTag(key: string, value: Primitive): void;
+```
+
+This function will allow user to add custom tags to current transaction for query. Primitive is a datatype which contains "number | string | boolean | bigint | symbol | null | undefined" value.
+
+### `errorHandler`
+
+``` typescript
+errorHandler(err): string | null;
+```
+
+This function will log details in monitoring application whenever it happens in the application.
+
+### `measurePerformance`
+
+``` typescript
+const capture = measurePerformance(name: string, op: string, data?: { [key: string]: number | string }): SentryTransactionObject;
+```
+
+Where name is the module name for which we want to measure performance. Operation is the functionality of the module and data is optional field that needed to be send to the transaction. It will return a object which contains different functions to use in current transaction. Here are the functions which will be return as an object.
+
+## SentryTransactionObject
+
+| **Config option**        | **Params**                                                   | **Description**                                                                              |
+|--------------------------|--------------------------------------------------------------|----------------------------------------------------------------------------------------------|
+| `setTag`                 | name: string, value: string                                  | Set tag in a transaction instance                                                            |
+| `startSpan`              | op: string, data: { [key: string]: number | string } | null  | Create a span in a transaction instance to measure the performance for a sub event           |
+| `finishSpan`             | op: string                                                   | Finish a running span in a transaction instance and complete the measurement for a sub event |
+| `finish`                 | status: string                                               | Finish a running transaction instance and complete the measurement for a main event          |
+
+User can set tags, create new span and finish span & transaction at the end to send measured data to sentry. `capture.finish('Ok')`
+
+### `setMeasurement`
+
+``` typescript
```

Comment:
Please remove the space between the triple backticks and the language keyword :

` ``` typescript` → ` ```typescript`

This is because in some markdown viewers / editors, the space will fail to assign the language.

Please do this for all 5 occurences :

- line 41
- line 49
- line 57
- line 65
- line 84